tests/remote-store.sh

```diff
@@ -5,8 +5,16 @@ clearStore
 # Ensure "fake ssh" remote store works just as legacy fake ssh would.
 nix --store ssh-ng://localhost?remote-store=$TEST_ROOT/other-store doctor
 
+# Ensure that check-trusted-user works with ssh-ng://
+nix --store ssh-ng://localhost?remote-store=$TEST_ROOT/other-store store check-trusted-user
+
 startDaemon
 
+# Ensure that check-trusted-user works with daemon
+if isDaemonNewer "2.13pre0"; then
```

Comment:
This can be updated, but fine if you don't bother fixing this until we are about to merge so you aren't stuck doing it again.

Reply:
This means the daemon needs to be newer than 2.13, so it is the correct value. When 2.13 was released, this was bumped from 2.12 to 2.13.